addons/l10n_lv/data/account.account.template.csv

```diff
@@ -0,0 +1,262 @@
+"a1520","Loans to related companies","1520","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a1530","Participation in the capital of associated companies","1530","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a1540","Loans to associated companies","1540","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a1550","Other securities and investments","1550","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a1560","Other loans and other long-term debtors","1560","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a1570","Own shares","1570","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a1580","Loans to shareholders or members and management","1580","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a1590","Deferred tax assets","1590","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a2110","Raw materials, basic materials and auxiliary materials","2110","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a2120","Unfinished production","2120","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a2130","Finished products and goods for sale","2130","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a2140","Uncompleted orders","2140","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a2150","Low-value inventory","2150","asset_fixed","l10n_lv.chart_template_latvia",,"False"
+"a2159","Write-down off low-value inventory","2159","asset_fixed","l10n_lv.chart_template_latvia",,"False"
+"a2190","Advance payments for goods","2190","Prepayments","l10n_lv.chart_template_latvia",,"False"
+"a2210","Animals","2210","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a2220","Young and small animals","2220","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a2230","Annual plantings","2230","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a2310","Accounts receivable – trade","2310","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2318","Settlement of doubtful debtors","2318","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2320","Bills of exchange received for deliveries and customers","2320","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2330","Debts of subsidiaries","2330","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2340","Debts of related companies","2340","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2350","Settlements with other debtors","2350","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2360","Settlement of unpaid amounts of the company's capital","2360","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2370","Short-term loans for company shareholder and management","2370","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2380","Settlement of claims on staff","2380","asset_receivable","l10n_lv.chart_template_latvia",,"True"
```

Comment:
Are really all of these receivables?
I would expect the "Settlement" accounts to be of another type :thinking: